specification/containerservice/Fleet.Management/.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+output
+tspconfig.yaml
```

Comment:
Shouldn't ignore tspconfig.yaml, but should have the same as what's currently in the main branch.